### PR TITLE
milliTimestamp added and postnum edited

### PR DESCRIPTION
**Pull request by mopsyshin**
problem solved

### CODE REVIEW

File: src/components/Main.vue [hunk 1]
```diff
@@ -148,15 +148,19 @@
         });
       },
       submit() {
-        let timestamp = Date();
+        var date = new Date();
+        var millisecond = moment().millisecond()
+        var timestamp = date
+        var milliTimestamp = `${date}${millisecond}`
```
Comment (jybaek) [inline src/components/Main.vue]: how about use `new Date().getTime()`

```node
> new Date().getTime()
1523952407531
>
```

File: src/components/Main.vue [hunk 2]
```diff
@@ -188,6 +192,14 @@
     },
     components: {
       Card: Card,
+    },
+    directives: {
```
Comment (jybaek) [inline src/components/Main.vue]: plz delete not use code

**Comment by mopsyshin** [comment]
another problem occured! ><
i'll solve it 1 hour later

File: src/components/Main.vue (hunk 2 above)
Comment (mopsyshin) [reply src/components/Main.vue]: gotit

**Comment by mopsyshin** [comment]
Yeah there is. Sorting problem

**Comment by mopsyshin** [comment]
error fixed

**Pull request merged**